src/en/02_quick_start.md

```diff
@@ -1,27 +1,50 @@
 # Quick Start
 
-## Install guide
+## Install Quide
 
-Download PgManage distribution file for your platform from the CommandPrompt website: https://www.commandprompt.com/products/pgmanage/
+Download PgManage distribution file for your platform from the [Command Prompt website](https://www.commandprompt.com/products/pgmanage/).
 
 ### Linux
 
-PgManage for Linux is packaged in .AppImage format and does not require installation. Download the .AppImage format, make it executable and run.
+PgManage for Linux is packaged in `.AppImage` format and does not require installation. Download the `.AppImage` format, make it executable, and run it. For example, in an Ubuntu distribution:
+
+```
+chmod +x pgmanage-1.0.0a.AppImage
+/pgmanage-1.0.0a.AppImage
```

Comment:
/pgmanage-1.0.0a.AppImage - the dot before the filename is missing here, it should look like ./pgmanage-1.0.0a.AppImage
Also, maybe we should avoid using exact file names in the guide? Maybe do something like this instead:
```
chmod +x pgmanage-$version.AppImage
./pgmanage-$version.AppImage
```